austrakka/utils/api.py

```diff
@@ -4,199 +4,169 @@
 from typing import List
 from typing import Union
 from json.decoder import JSONDecodeError
+from http import HTTPStatus
 
-from loguru import logger
-import requests
-import click
-from requests_toolbelt.multipart.encoder import MultipartEncoder
-from requests.exceptions import HTTPError
+from httpx import HTTPStatusError
+import httpx
 
-from austrakka.utils.enums.api import RESPONSE_TYPE
-from austrakka.utils.enums.api import RESPONSE_TYPE_ERROR
 from austrakka.utils.exceptions import FailedResponseException
 from austrakka.utils.exceptions import UnknownResponseException
-from ..components.auth.enums import Auth
-from .misc import logger_wraps
-from .output import log_dict
-from .output import log_response
+from austrakka.components.auth.enums import Auth
+from austrakka.utils.output import log_response
+from austrakka.utils.context import CxtKey
+from austrakka.utils.context import get_ctx_value
+from austrakka import __version__
 
-get = requests.get
-post = requests.post
-put = requests.put
-patch = requests.patch
+CONTENT_TYPE_JSON = 'application/json'
+CONTENT_TYPE_MULTIPART = 'multipart/form-data; charset=utf-8; boundary=+++'
 
-NO_CONTENT = 204
-TIMEOUT_IN_SECONDS = 300
 
-requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+def _get_default_headers(
+        content_type: str = CONTENT_TYPE_JSON,
+) -> Dict:
+    default_headers = {
+        'Content-Type': content_type,
+        'Authorization': f'Bearer {get_ctx_value(CxtKey.CTX_TOKEN)}',
+        'Ocp-Apim-Subscription-Key': Auth.SUBSCRIPTION_KEY.value,
```

Comment:
Remove Apim sub key

Reply:
Sure, I had a separate PR but I may as well bundle it here